### ISSUE TITLE
Rename Logit loss

### ISSUE THREAD

**Issue by maurapintor** [pralab/secml-torch#10]
Rename "Logits loss" to "Logit Loss"

https://github.com/pralab/secml2/blob/bbaa69c0c61b9c79618da414b51ae88462663db1/src/adv/evasion/composite_attack.py#L16
